replace.py

```diff
@@ -0,0 +1,219 @@
+"""
+/******************************************************************************
+  This source file is part of the Avogadro project.
+
+  This source code is released under the New BSD License, (the "License").
+******************************************************************************/
+"""
+
+import argparse
+import json
+import sys
+import random
+
+# Some globals:
+debug = True
+
+element_dict = {
+    "Hydrogen": 1,
```

Comment:
This is fine. You could actually have just used a list and changed the index as needed (e.g., elements[0] => "Hydrogen" .. which is atomic number 1). But that's a minor point.